.github/workflows/terragrunt-plan-staging.yml

```diff
@@ -25,6 +25,9 @@ env:
   TF_VAR_rds_db_password: ${{ secrets.STAGING_DB_PASSWORD }}
   TF_VAR_slack_webhook: ${{ secrets.STAGING_SLACK_WEBHOOK }}
   TF_VAR_gc_notify_callback_bearer_token: ${{ secrets.STAGING_GC_NOTIFY_CALLBACK_BEARER_TOKEN }}
+  TF_VAR_gc_notify_url: https://notification.canada.ca
```

Comment:
This is not the correct value. It should be https://api.notification.canada.ca/ . I would recommend hard coding this value since its not changingThis is not the correct value. It should be https://api.notification.canada.ca/ . I would recommend hard coding this value since its not changing